nautobot_floor_plan/templates/nautobot_floor_plan/location_child_floor_plan.html

```diff
@@ -0,0 +1,18 @@
+{% extends "dcim/location.html" %}
+{% load helpers %}
+{% load static %}
+{% load seed_helpers %}
+{% block content %}
+<div class="panel panel-default">
+    <div class="panel-heading">
+        <strong>Child Floor Plans</strong>
+        <div class="pull-right noprint">
+            {% if request.user.is_authenticated %}
```

Comment:
Is there a reason we are checking if the user is authenticated here? This seems out of place or copy pasta from somewhere else.

Reply:
not needed, will remove